### PR TITLE
feat(developer): Project Settings Form for 17.0+ projects 🦕

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to #9948.

# User Testing

* **TEST_SETTINGS_FORM:** Upgrade an existing project to the new Keyman Developer 17.0 project format. Open the Project Settings dialog, and go through and change each setting, then check that each setting is saved when you click OK and then re-open the dialog.

### CODE REVIEW

File: developer/src/tike/project/Keyman.Developer.UI.Project.UfrmProjectSettings20.pas [hunk 1]
```diff
@@ -0,0 +1,82 @@
+(*
+  Name:             Keyman.Developer.UI.Project.UfrmProjectSettings
```
Comment (darcywong00) [inline developer/src/tike/project/Keyman.Developer.UI.Project.UfrmProjectSettings20.pas:2]:
```suggestion
  Name:             Keyman.Developer.UI.Project.UfrmProjectSettings20
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by mcdurdin** [comment]
# Test Results

* **TEST_SETTINGS_FORM (PASSED):** Checked each field and all settings were correctly persisted

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.211-alpha](https://keyman.com/downloads/releases/alpha/17.0.211)

**Pull request merged**